.gitignore

```diff
@@ -66,3 +66,4 @@ test/
 frontend/__history/
 frontend/logs/
 frontend/patches/
+*.txaPackage
```

Comment:
This is a large decision.  I can see how it'd make sense to let the IDE generate these per-machine, but I could also see how it might want to be preserved because I think they can store user modifications.  I suppose it depends on whether or not we have any plans of ever using the RAD Studio automatic diagrams for anything (I personally don't think I will, I prefer making my own diagrams with Visual Paradigm).